core/src/main/scala/knobs/Parser.scala

```diff
@@ -38,22 +38,22 @@ object ConfigParser {
   }
 
   /** Top-level parser of configuration files */
-  lazy val topLevel: Parser[List[Directive]] = "configuration" |: {
-    directives << skipLWS << realEOF
-  }
+  lazy val topLevel: Parser[List[Directive]] = topLevel(withImport = true)
```

Comment:
I'm not 100% sure what you mean as I don't see anything being shadowed. Are you saying you are not a fan of having the value `topLevel` and the definition `topLevel` share the same name? That kind of thing doesn't seem to bother me all that much, I guess I haven't been sufficiently burned by doing it yet. I guess I did it to avoid using a default value for `withImport`, not sure why I did that. I'm happy to rename one of the two or give `withImport` a default value. Do you have a suggestion?